packages/armbian/builddeb

```diff
@@ -65,25 +65,6 @@ create_package() {
 	# in case we build in a setuid/setgid directory
 	chmod -R ug-s "$pdir"
 
-	# Create preinstall and post install script to remove dtb
```

Comment:
I would propose to move this whole part of changing the way DTBs are cleaned (all from this poing down) to separate PR and we deal with it another day.